content/scout/guides/vex.md

```diff
@@ -0,0 +1,345 @@
+In-toto attestations already contain the image reference in the attestation predicate,
+making the image reference in the `products` key of the VEX document redundant.
+Instead, `products` should refer to the packages that contain the vulnerabilities
+(`subcomponents` in the VEX document shown earlier).
+
+1. Create a new `in-toto.vex.json` document with the following contents:
+
+   ```json
+   {
+     "@context": "https://openvex.dev/ns/v0.2.0",
+     "@id": "https://openvex.dev/docs/public/vex-a7e7c69be57bc49dec9cc2f3a3e3329b12674ca53b53a53ab42134dcc0510779",
+     "author": "author@example.com",
+     "timestamp": "2024-02-03T09:33:28.913572+01:00",
+     "version": 1,
+     "statements": [
+       {
+         "vulnerability": {
+           "name": "CVE-2022-24999"
+         },
+         "timestamp": "2024-02-03T09:33:28.913574+01:00",
+         "products": [
+           {
+             "@id": "pkg:npm/express@4.17.1"
+           },
+           {
+             "@id": "pkg:npm/qs@6.7.0"
+           }
+         ],
+         "status": "not_affected",
+         "justification": "vulnerable_code_not_in_execute_path"
+       }
+     ]
+   }
+   ```
+
+2. Attach the `in-toto.vex.json` VEX document as an attestation:
+
+   ```console
+   $ docker scout attestation add \
+     --file in-toto.vex.json \
+     --predicate-type https://openvex.dev/ns/v0.2.0 \
+     <ORG>/scout-demo-service:v1
+   ```
+
+   This adds an in-toto attestation to the image,
+   with a predicate type of `https://openvex.dev/ns/v0.2.0`.
+
+3. Analyze the image with `docker scout cves`.
+
+   > **Note**
+   >
+   > This only works when analyzing remote images in a registry.
+   > To force Docker Scout to analyze a registry image instead of a local one,
+   > specify the image reference with a `registry://` prefix.
+
+   ```console
+   $ docker scout cves \
+     --only-cve-id CVE-2022-24999 \
+     registry://<ORG>/scout-demo-service:v1
+   ```
+
+## Create a VEX advisory
+
+To apply VEX statements when analyzing images
+without having to specify local documents or attach them as attestations,
+you can create a VEX advisory for your Docker organization.
+
+When Docker Scout analyzes an image,
+it checks for the existence of a VEX advisory in your GitHub organization.
+The advisory is a GitHub repository named `scout-advisories`,
+containing a `vex` directory with VEX documents inside.
+If an advisory exists, any VEX documents inside are automatically applied.
+The only prerequisite is that you enable the GitHub integration for the repository
+in the Docker organization where you've enabled Docker Scout.
+
+To create a VEX advisory:
+
+1. Create a new GitHub repository named `docker-scout-advisories`.
+2. Clone the repository to your machine.
+3. Create a `vex` directory in the `scout-advisories` repository.
+4. Copy the `CVE-2022-24999.vex.json` file to `scout-advisories/vex/CVE-2022-24999.vex.json`.
+5. Commit to the default branch (e.g. `main`) and push GitHub.
```

Comment:
```suggestion
5. Commit to the default branch (e.g. `main`) and push to GitHub.
```

Reply:
Deleting this section as it's descoped for now